Scripts/RemediationScripts/Remediate-DisableNonSSLPortOnRedisCache.ps1

```diff
@@ -363,12 +445,14 @@ function Disable-NonSSLPortOnRedisCache
             if($userInput -ne "Y")
```

Comment:
if(-not force) one should have an wrapper condtion of if(-not $Autoremediation) as for autoremediation feature we won't be asking input from user again